doc/api/domain.md

```diff
@@ -115,7 +115,7 @@ if (cluster.isMaster) {
     d.on('error', (er) => {
       console.error(`error ${er.stack}`);
 
-      // Note: we're in dangerous territory!
+      // Note: We're in dangerous territory!
```

Comment:
**Note:** that's falls outside the scope of this PR, but "we" shouldn't have been used here at all. I'll reword all of the comments in this example in another PR.